collector/mongodb_collector.go

```diff
@@ -130,6 +132,10 @@ func (exporter *MongodbCollector) Collect(ch chan<- prometheus.Metric) {
 			glog.Info("Collecting Connection Pool Stats")
 			exporter.collectConnPoolStats(mongoSess, ch)
 		}
+		if exporter.Opts.CollectParameterMetrics {
+			glog.Info("Collection parameter metrics")
+			exporter.collectParameter(mongoSess, ch, exporter.Opts.CollectParameters)
```

Comment:
These options should be split here and passed to the collector as a slice of strings. This way we can validate that there is at least one, trim spaces, etc. all with the option of stopping startup due to an error.

Reply:
This won't cause a startup error, this gets run after startup